src/components/grid/grid-test.stories.tsx

```diff
@@ -21,25 +22,20 @@ export const GridStory = ({
   item31300,
   item31500,
   item3900,
-  ...args
-}) => {
-  const group = "GridItem";
-  const viewportSettings = "viewport settings";
-  const groupID1 = `${group} 1`;
-  const groupID2 = `${group} 2`;
-  const groupID3 = `${group} 3`;
+}: // eslint-disable-next-line @typescript-eslint/no-explicit-any
+any) => {
```

Comment:
Instead of `any` could we not define the needed interfaces?

```
interface ItemProps {
  alignSelf: string;
  gridColumn: string;
  gridRow: string;
  justifySelf: string;
  maxWidth: string;
}

interface Items {
  [key: string]: ItemProps; // you could probably define each item name but it might be overkill
}

export const Default = ({
  item11500,
  item11300,
  item1900,
  item21300,
  item21500,
  item2900,
  item31300,
  item31500,
  item3900,
}: Items)
```